ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.java

```diff
@@ -134,7 +136,18 @@ public void setBackground(Drawable drawable) {
         "This method is not supported for ReactViewGroup instances");
   }
 
-  public void setTranslucentBackgroundDrawable(@Nullable Drawable background) {
+  public void setNativeBackground(@Nullable ReadableMap nativeBackground) {
+    mNativeBackground = nativeBackground;
+    refreshTranslucentBackgroundDrawable();
+  }
+
+  private void refreshTranslucentBackgroundDrawable() {
+    Drawable background = null;
+    if (mNativeBackground != null) {
+      float[] cornerRadii = getOrCreateReactViewBackground().getBorderRadii();
```

Comment:
I think you may want to check if `mReactBackgroundDrawable != null` first. Otherwise this line will create that background object for every view with native background which is undesirable